.github/workflows/periodic.yml

```
@@ -32,6 +32,12 @@ jobs:
     if: github.repository_owner == 'submariner-io'
     runs-on: ubuntu-latest
     steps:
+      - name: Setup kernel, increase the inotify settings
+        run: |
+          echo -e "fs.inotify.max_user_watches=524288\nfs.inotify.max_user_instances=512" \
+          | sudo tee -a /etc/sysctl.conf && \
+          sudo sysctl -p
```

No need to write a file:

```suggestion
          sudo sysctl -w fs.inotify.max_user_watches=524288 fs.inotify.max_user_instances=512
```